docs/index.rst

```diff
@@ -1426,6 +1425,21 @@ Setup |ccloud|
 
       ccloud login --save
 
+#. At this point, you have two options for running the |ccloud| portion of this tutorial.
+
+   The first option is to manually complete all the steps in the following sections, which we recommend if you are new to |ccloud| and |cp|: :ref:`cp-demo-ccloud-stack`, :ref:`cp-demo-telemetry-reporter`, :ref:`cp-demo-replicator-to-ccloud`, and :ref:`cp-demo-metrics-api`.
```

Comment:
Should the `Cleanup` section be mentioned here?

Reply:
@mikebin agree, and fixed.  Thank you.